packages/webpack-cli/lib/groups/AdvancedGroup.js

```diff
@@ -22,11 +22,10 @@ class AdvancedGroup extends GroupHelper {
         if (args.prefetch) {
             const { PrefetchPlugin } = require('webpack');
             const prefetchVal = new PrefetchPlugin(null, args.prefetch);
-            if (options && options.plugins) {
```

Comment:
Accessing `options.plugin` would raise error if `options` is undefined. It should be reverted imo.

Reply:
on it